Shared/parameters.json

```diff
@@ -48,6 +48,9 @@
         "keyVaultName": {
             "value": "__keyVaultName__"
         },
+        "keyVaultResourceGroup": {
```

Comment:
What do we need this additional parameter for?

Reply:
In case the key vault is not in the same resource group as the service being deployed (e.g. red service using the shared kv). It could be a default value on the parameter instead?

Reply:
Cool